bin/server

```diff
@@ -51,10 +51,19 @@ const executor = new ExecutorPlugin({
 });
 
 // Source Code Plugin
-const scmConfig = config.get('scm');
-const ScmPlugin = require(`screwdriver-scm-${scmConfig.plugin}`);
-const scm = new ScmPlugin(hoek.applyToDefaults({ ecosystem },
-    (scmConfig[scmConfig.plugin] || {})));
+let scm;
+
+if (config.has('scms')) {
+    const scmConfigs = config.get('scms');
+    const ScmPlugin = require('screwdriver-scm-router');
+    scm = new ScmPlugin(hoek.applyToDefaults({ ecosystem },
+        { scms: scmConfigs }));
+} else {
+    const scmConfig = config.get('scm');
+    const ScmPlugin = require(`screwdriver-scm-${scmConfig.plugin}`);
+    scm = new ScmPlugin(hoek.applyToDefaults({ ecosystem },
```

Comment:
`scm-router` can handle a scm module set by `scm`. Can not use without using `if`? 🤔 
https://github.com/wahapo/scm-router-1/blob/initial/index.js#L33